src/v/cloud_storage/cache_service.cc

```diff
@@ -880,7 +880,18 @@ ss::future<> cache::stop() {
     _block_puts_cond.broken();
     _cleanup_sm.broken();
     if (ss::this_shard_id() == 0) {
-        co_await save_access_time_tracker();
+        co_await save_access_time_tracker().handle_exception([](auto eptr) {
```

Comment:
Just double checking: The intent is not just to log but also catch it (and avoiding propagation)?

Reply:
Yes. I think it's appropriate in this context because it's in the shutdown phase. It's better to log an error and see if this happens in a real redpanda cluster.
The unit test itself succeded, it's the unit test process that's failing because of this uncaught exception and this is spoiling the unit test result